static/bidder-params/bidmachine.json

```diff
@@ -0,0 +1,31 @@
+{
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "Kidoz Adapter Params",
```

Comment:
Typo: Please change title and description to reference bidmachine instead of kidoz.